server/pom.xml

```diff
@@ -744,7 +744,10 @@
       <dependency>
         <groupId>org.zanata</groupId>
         <artifactId>zanata-common-api</artifactId>
+        <!-- release plugin does not play nice with variable substitution
         <version>${zanata.api.compat.version}</version>
+        -->
```

Comment:
If we only use api-compat in one place, let's just delete the comment and the variable.